src/test/resources/features/ifs/fieldCollectionsIFS/VATFC.feature

```diff
@@ -2,175 +2,153 @@ Feature: FC VAT Debt Calculation End point testing
 
   Scenario: 1. Interest Indicator as Yes. 1 Payment of 1 debt.
     Given a fc vat debt item
-      | debtId | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
-      | 123    | 500000         | 2018-12-16 | 2019-04-14          | Y                 |
+      | debtItemChargeId  | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
+      | debtItemChargeId1 | 500000         | 2018-12-16 | 2022-01-04          | Y                 |
     And the fc vat debt item has payment history
       | paymentAmount | paymentDate |
       | 100000        | 2019-02-03  |
-    And the fc vat customer has breathing spaces applied
-      | debtRespiteFrom | debtRespiteTo |
-      | 2020-04-06      | 2021-04-06    |
+    And no breathing spaces have been applied to the fc vat customer
     When the debt item is sent to the fc vat ifs service
     Then the fc vat ifs service wilL return a total debts summary of
       | combinedDailyAccrual | unpaidAmountTotal |
-      | 0                    | 400000            |
-#    Uncomment and fix step below when DTD-1023 has been fixed
-#    And the 1st fc vat debt summary will contain
-#      | debtId | interestDailyAccrual | interestRate |
-#      | 123    | 28                   | 2.75         |
+      | 30                   | 400000            |
+    And the 1st fc vat debt summary will contain
+      | debtItemChargeId  | interestDueDailyAccrual | interestRate |
+      | debtItemChargeId1 | 30                      | 2.75         |
 
 
   Scenario: 2. Interest Indicator as No. 1 Payment of 1 debt.
     Given a fc vat debt item
-      | debtId | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
-      | 123    | 500000         | 2018-12-16 | 2019-04-14          | N                 |
+      | debtItemChargeId  | originalAmount | periodEnd  | interestRequestedTo | interestIndicator |
+      | debtItemChargeId1 | 500000         | 2018-12-16 | 2019-04-14          | N                 |
     And the fc vat debt item has payment history
       | paymentAmount | paymentDate |
       | 100000        | 2019-02-03  |
-    And the fc vat customer has breathing spaces applied
-      | debtRespiteFrom | debtRespiteTo |
-      | 2020-04-06      | 2021-04-06    |
+    And no breathing spaces have been applied to the fc vat customer
     When the debt item is sent to the fc vat ifs service
     Then the fc vat ifs service wilL return a total debts summary of
       | combinedDailyAccrual | unpaidAmountTotal |
       | 0                    | 400000            |
-#    Uncomment and fix step below when DTD-1023 has been fixed
-#    And the 1st fc vat debt summary will contain
-#      | debtId | interestDailyAccrual | interestRate |
-#      | 123    | 28                   | 2.75         |
+    And the 1st fc vat debt summary will contain
```

Comment:
Atm this bug https://jira.tools.tax.service.gov.uk/browse/DTD-1022 is not in development and therefore these tests will most likely fail. Would it be better to add these in a separate pr for the other bug?

Reply:
@vinniebrice they're passing based on the IFS DTD-871 branch looks like DTD-1022 might not be needed

Reply:
Awesome!